.github/workflows/pull-requests.yml

```diff
@@ -33,50 +33,24 @@ jobs:
             fail-fast: false
             matrix:
                 type:
-                    - circuits
                     - contracts
                     - libraries
 
         steps:
             - name: Checkout
               uses: actions/checkout@v4
 
-            - name: Get changed files
-              id: changed-files-contracts
-              uses: tj-actions/changed-files@v44
-              with:
-                  files: |
-                      packages/**.sol/**
-
             - name: Setup Node
               uses: actions/setup-node@v4
               with:
                   node-version: "20"
                   cache: yarn
 
-            # https://github.com/iden3/circuits/blob/8fffb6609ecad0b7bcda19bb908bdb544bdb3cf7/.github/workflows/main.yml#L18-L22
-            # https://stackoverflow.com/a/78377916
-            - name: Setup Circom deps
-              if: matrix.type != 'libraries'
-              run: |
-                  sudo rm /etc/apt/sources.list.d/microsoft-prod.list
-                  sudo apt-get update && sudo apt-get install -y wget nlohmann-json3-dev libgmp-dev nasm g++ build-essential
-
-            - name: Setup Circom
-              run: wget https://github.com/iden3/circom/releases/latest/download/circom-linux-amd64 && sudo mv ./circom-linux-amd64 /usr/bin/circom && sudo chmod +x /usr/bin/circom
-
-            - name: Install Nargo
-              uses: noir-lang/noirup@v0.1.3
-              with:
-                  toolchain: 0.26.0
-
             - name: Install dependencies
               run: yarn
 
             - name: Build libraries and contracts
               run: yarn build
 
             - name: Test ${{ matrix.type }}
-              # this condition is to skip contract tests when PR changes are unrelated
```

Comment:
Removed as contracts depends on local JS libs. If there's a change in the `@zk-kit/imt` lib, it'd be better to test contracts too.

Reply:
makes sense for now!